packages/test/test-end-to-end-tests/src/test/gc/gcTombstone.spec.ts

```diff
@@ -564,6 +563,41 @@ describeNoCompat("GC tombstone tests", (getTestObjectProvider) => {
             sendDataObject._runtime.submitSignal("can receive", "a signal");
             await provider.ensureSynchronized();
         });
+
+        itExpects("does not throw tombstone errors when ThrowOnTombstoneUsage setting is not enabled",
+        [
+            { eventName: "fluid:telemetry:Summarizer:Running:SweepReadyObject_Loaded" },
+            { eventName: "fluid:telemetry:FluidDataStoreContext:GC_Tombstone_DataStore_Changed", callSite: "submitMessage" },
+            { eventName: "fluid:telemetry:FluidDataStoreContext:GC_Tombstone_DataStore_Changed", callSite: "process" },
+            { eventName: "fluid:telemetry:ContainerRuntime:GC_Tombstone_DataStore_Requested" },
+        ],
+        async () => {
+            settings["Fluid.GarbageCollection.ThrowOnTombstoneUsage"] = false;
+            const {
+                unreferencedId,
+                summarizingContainer,
+                summarizer,
+            } = await summarizationWithUnreferencedDataStoreAfterTime(sweepTimeoutMs);
+
+            // Use the request pattern to get the testDataObject - this is unsafe and no one should do this in their
+            // production application - causes a sweep ready loaded error
+            const dataObject = await requestFluidObject<ITestDataObject>(summarizingContainer, unreferencedId);
```

Comment:
Reminder for myself, this test will need to be updated since you're creating the dataObject from the summarizingContainer. We will need to make sure we are loading a dataObject in a non-summarizer client from the summary.